terraform/README.md

```diff
@@ -0,0 +1,16 @@
+## Decal VM Terraform configuration
+
+There are two additional files containing sensitive
+information to use with this configuration,
+credentials.auto.tfvars and students.auto.tfvars.
+
+The .tfstate file is also available via more secure channels.
```

Comment:
Same for adding an explanation of what this file does and why it is kept private.